### PR TITLE
Added some missing CNAME trackers

### PR DESCRIPTION
**Pull request by ameshkov**

### CODE REVIEW

**Comment by spirillen** [comment]
I can confirm them all as adware/tracking.

File: domains [hunk 1]
```diff
@@ -6,6 +6,16 @@
 # wildcard match (domain and all its multi-level subdomains) CNAMEs
 # against the domains in this list.
 
+# Pardot
+go.pardot.com
```
Comment (spirillen) [inline domains:10]: If NextDNS does support wildcard blocking, there should be no reason not to do this here.

The entire domain is used for [Tracking](https://www.mypdns.org/w/trackware/) & [Adware](https://www.mypdns.org/w/adware/)

See my full RPZ history here: <https://www.mypdns.org/T3868>

File: domains [hunk 2]
```diff
@@ -6,6 +6,16 @@
...
+
+# Act-On
+actonservice.com
+actonsoftware.com
+
+# Oracle Eloqua
+hs.eloqua.com
```
Comment (spirillen) [inline domains:17]: Same as above, Full RPZ history of blocking: <https://www.mypdns.org/T3873>

**Comment by romaincointepas** [comment]
Added in https://github.com/nextdns/cname-cloaking-blocklist/commit/06336c121b641d4d954bf38ef51ca97cf9ee48e1 👍

**Pull request closed without merging**